iwyu.cc

```diff
@@ -2386,7 +2386,7 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
         using clang::DirectoryLookup;
         using clang::FileEntryRef;
         const FileEntry* use_file = CurrentFileEntry();
-        const DirectoryLookup* curdir = nullptr;
+        const DirectoryLookup** curdir = nullptr;
```

Comment:
Let's just pass nullptr, no need for the named variable. That was the rationale for the change as described in the commit message here: https://github.com/llvm/llvm-project/commit/105c913156e94163c937d812992fe3a56210998c.

Reply:
Oh, and you can also get rid of the `using clang::DirectoryLookup;` line in that case.